modules/host/errors_test.go

```diff
@@ -0,0 +1,330 @@
+package host
+
+import (
+	"bufio"
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/modules"
+)
+
+// countFileLines is a helper function that will count the number of lines in a
+// file, based on the number of '\n' characters. countFileLines will load the
+// file into memory using ioutil.ReadAll.
+//
+// countFileLines will ignore all lines with the string 'DEBUG' in it.
+func countFileLines(filepath string) (uint64, error) {
```

Comment:
no but it might screw something else up, especially if you don't replace all of the methods. A logger is a type, not an interface, which makes it more difficult to swap out without side effects.

It probably wouldn't be too bad in this case, but the code to do it without mocking is already in place, and imo if you don't need to mock it's better to get your coverage without it.